src/components/Clock.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  today: Date,
+};
+
+export class Clock extends React.Component<Props> {
+  state: State = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Readonly<Props>): void {
+    if (prevProps.clockName !== this.props.clockName) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProps.clockName} to ${this.props.clockName}`);
+    }
+  }
```

Comment:
your code is not working because of this
![image](https://github.com/mate-academy/react_clock/assets/133110903/1996bc29-2305-4a5e-892e-cb0277df9a96)